gpu/compute.go

```diff
@@ -1898,11 +1898,11 @@ func (c *collector) hashOp(op paintOp) uint64 {
 	for _, cl := range op.clipStack {
 		k := cl.state
 		keyBytes := (*[unsafe.Sizeof(k)]byte)(unsafe.Pointer(unsafe.Pointer(&k)))
-		c.hasher.Write(keyBytes[:])
+		_, _ = c.hasher.Write(keyBytes[:])
```

Comment:
Revert these changes. FWIW, I don't agree the extra ceremony `_, _` adds is worth it.

Reply:
They silence the "error return" warnings from `golangci-lint` for those lines. The remaining warnings, about a dozen, seem to require error checks.

Also I do recommend to add this linter to your pipeline.

Reply:
We appreciate your recommendation, but we choose not to use that metalinter at this time. Many of the linters that it executes are unhelpful, and some of them produce contradictory recommendations. `golangci-lint` also disables some important checks within `staticcheck`. Silencing a lint warning from a linter that we do not use is not sufficient reason to make this change.

Reply:
fair enough